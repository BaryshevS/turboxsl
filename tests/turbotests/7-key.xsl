<?xml version="1.0" encoding="UTF-8"?>
<xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform" xmlns:fo="http://www.w3.org/1999/XSL/Format" xmlns:chk="LTR:chk" xmlns:ltr="LTR" exclude-result-prefixes="ltr chk" xmlns:exsl="http://exslt.org/common" extension-element-prefixes="exsl">

<xsl:key name="title-search" match="book" use="@author"/>
<xsl:template match="/">
	<head>
		<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
	</head>
	<body>
		<xsl:for-each select="key('title-search', 'David Perry')">
				 <xsl:value-of select="@title"/><br/>
      </xsl:for-each>
	</body>
</xsl:template>

</xsl:stylesheet>
