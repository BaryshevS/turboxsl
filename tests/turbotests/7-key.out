<?xml version="1.0"?>
<head xmlns:fo="http://www.w3.org/1999/XSL/Format"><meta http-equiv="Content-Type" content="text/html; charset=utf-8"/></head><body xmlns:fo="http://www.w3.org/1999/XSL/Format">XML Today<br/>XML and Microsoft<br/></body>
